idaes/models_extra/power_generation/costing/power_plant_capcost.py

```diff
@@ -0,0 +1,1995 @@
+
+        # pull data for each account into dictionaries
+        process_params = {}
+        reference_units = {}
+        account_names = {}
+        exponents = {}
+        reference_costs = {}
+        reference_cost_units = {}
+        reference_costs_init = {}
+        reference_params = {}
+        cost_scaling_fractions = {}
+        engineering_fees = {}
+        process_contingencies = {}
+        project_contingencies = {}
+
+        for account in cost_accounts:
+            try:  # first look for data in json file info
+                process_params[account] = BB_costing_params[str(tech)][ccs][account][
+                    "Process Parameter"
+                ]
+                reference_units[account] = BB_costing_params[str(tech)][ccs][
+                    cost_accounts[0]
+                ]["Units"]
+                account_names[account] = BB_costing_params[str(tech)][ccs][account][
+                    "Account Name"
+                ]
+                exponents[account] = float(
+                    BB_costing_params[str(tech)][ccs][account]["Exponent"]
+                )
+                reference_costs[account] = BB_costing_params[str(tech)][ccs][account][
+                    "BEC"
+                ]
+                reference_cost_units[account] = BB_costing_params[str(tech)][ccs][
+                    account
+                ]["BEC_units"]
+                reference_costs_init[account] = (
+                    BB_costing_params[str(tech)][ccs][account]["BEC"] * 1e-3
+                )
+
+                if type(process_params[account]) == list:
+                    for i, processparam in enumerate(process_params[account]):
+                        reference_params[account, processparam] = BB_costing_params[
+                            str(tech)
+                        ][ccs][account]["RP Value"][i]
+                        cost_scaling_fractions[
+                            account, processparam
+                        ] = BB_costing_params[str(tech)][ccs][account][
+                            "Cost scaling fraction"
+                        ][
+                            i
+                        ]
+
+                elif type(process_params[account]) == str:
+                    reference_params[account] = BB_costing_params[str(tech)][ccs][
+                        account
+                    ]["RP Value"]
+
+                engineering_fees[account] = BB_costing_params[str(tech)][ccs][account][
+                    "Eng Fee"
+                ]
+                process_contingencies[account] = BB_costing_params[str(tech)][ccs][
+                    account
+                ]["Process Contingency"]
+                project_contingencies[account] = BB_costing_params[str(tech)][ccs][
+                    account
+                ]["Project Contingency"]
+            except KeyError:
+                try:  # next look for data in custom dictionaries
```

Comment:
If it is possible to merge the original and custom dictionaries we will only need to look in one location and can cut down on duplicated code.

Reply:
I like this suggestion, as long as we don't mind keeping the bituminous baseline and cansolv/mea accounts in the same place. @MAZamarripa what are your thoughts on this? We could wait until the cleanup to merge them, since we will remove the old dictionaries at that point as well.

Reply:
I prefer to keep them separated: i) as an example to use "custom" costing correlations, ii) to keep "baseline" costing correlations isolated from others (custom, private, generic, public, etc.)

Reply:
I don't think we need to store all the accounts in the same place to do this. Earlier in this method we move entries from `additional_costing_params` to `custom_costing_params`. Could we then move the entries from `custom_costing_params` into `process_params`?

Reply:
Yes, that is possible, thank you for the suggestion @AlexNoring . I will update the method accordingly.